src/renderer_mtl.mm

```diff
@@ -655,39 +715,32 @@ bool init(const Init& _init)
 			// It is decremented by 1 because 1 entry is used for uniforms.
 			g_caps.limits.maxComputeBindings = bx::uint32_min(30, BGFX_MAX_COMPUTE_BINDINGS);
 
-			m_hasPixelFormatDepth32Float_Stencil8 = false
-				||  BX_ENABLED(BX_PLATFORM_OSX)
-				|| (BX_ENABLED(BX_PLATFORM_IOS) && iOSVersionEqualOrGreater("9.0.0") )
-				;
-
-			m_hasStoreActionStoreAndMultisampleResolve = false
-				|| (BX_ENABLED(BX_PLATFORM_OSX) && macOSVersionEqualOrGreater(10, 12, 0) )
-				|| (BX_ENABLED(BX_PLATFORM_IOS) && iOSVersionEqualOrGreater("10.0.0") )
-				;
-
-			m_macOS11Runtime = true
-				&& BX_ENABLED(BX_PLATFORM_OSX)
-				&& macOSVersionEqualOrGreater(10, 11, 0)
-				;
-
-			m_iOS9Runtime = true
-				&& BX_ENABLED(BX_PLATFORM_IOS)
-				&& iOSVersionEqualOrGreater("9.0.0")
-				;
+			m_hasPixelFormatDepth32Float_Stencil8 = atLeastOSVersion(macOSVersion(0), iOSVersion(9, 0, 0));
+			m_hasStoreActionStoreAndMultisampleResolve = atLeastOSVersion(macOSVersion(10, 12, 0), iOSVersion(10, 0, 0));
 
 			if (BX_ENABLED(BX_PLATFORM_OSX) )
 			{
 				s_textureFormat[TextureFormat::R8].m_fmtSrgb  = MTLPixelFormatInvalid;
 				s_textureFormat[TextureFormat::RG8].m_fmtSrgb = MTLPixelFormatInvalid;
 			}
 
+			if (!atLeastOSVersion(macOSVersion(11, 0, 0), iOSVersion(8, 0, 0))
+			||  (g_caps.vendorId == BGFX_PCI_ID_AMD))
```

Comment:
These texture formats were unsupported for me, on my AMD GPU (AMD Radeon R9 M370X 2 GB). As Apple doesn't document anything about AMD GPUs in their capability PDF, I just exclude them here for all AMD GPUs (assuming it's a driver issue).

Reply:
I looked around the docs, but couldn't find anything. They only refer to their PDF table... If you know how to, please let me know such that I can integrate and test it.